app/renderer/components/main/notificationBar.js

```diff
@@ -26,19 +27,24 @@ class NotificationBar extends React.Component {
     const notifications = state.get('notifications', Immutable.List())
 
     const props = {}
+    // TODO: Make use of getActiveTabNotifications API instead
     props.activeNotifications = notifications
       .filter((item) => {
         return item.get('frameOrigin')
           ? activeOrigin === item.get('frameOrigin')
-          : true
+          : !item.get('greeting') // greetings indicate a global notification
```

Comment:
this may break password saving notifications. those do *not* have a frameOrigin (see https://github.com/brave/browser-laptop/issues/8901#issuecomment-301991642), but they are also not greeting messages.

Reply:
ya but if they do not have greetings the condition will evaluate to `true` and it would be visible as-is now. You can assert by running `npm run test -- --grep="shows notification for login form"`. Let me know if I missed something

Reply:
ya but if no `greetings` then it will evaluate to `true` as well. Anyway, your call is correct and the check I was doing for per tab notifications were wrong and I updated. Thanks for checking

You can assert now it works by running npm run test -- --grep="shows notification for login form". Let me know if I missed something